versions.props

```diff
@@ -7,24 +7,24 @@ com.google.guava:guava = 23.6.1-jre
 com.jcraft:jzlib = 1.1.3
 com.netflix.concurrency-limits:* = 0.1.3
 com.netflix.feign:feign-*= 8.18.0
-com.palantir.conjure.java:conjure-java = 2.4.0
-com.palantir.conjure.java:conjure-lib = 2.4.0
+com.palantir.conjure.java:conjure-java = 2.5.0
+com.palantir.conjure.java:conjure-lib = 2.5.0
 com.palantir.conjure.java.api:* = 2.0.0
 com.palantir.conjure.verification:* = 0.15.0
 com.palantir.safe-logging:* = 1.5.1
 com.palantir.tracing:* = 1.2.0
 com.palantir.tritium:tritium-registry = 0.8.0
 com.squareup.okhttp3:* = 3.11.0
 com.squareup.retrofit2:* = 2.4.0
-io.dropwizard.metrics:metrics-core = 3.2.5
-io.dropwizard:dropwizard-* = 1.3.1
+io.dropwizard.metrics:metrics-core = 3.2.6
+io.dropwizard:dropwizard-* = 1.3.7
 io.reactivex.rxjava2:rxjava = 2.2.3
 javax.ws.rs:javax.ws.rs-api = 2.0.1
 junit:junit = 4.12
 org.apache.commons:commons-lang3 = 3.8.1
 org.assertj:assertj-core = 3.11.1
 org.bouncycastle:bcpkix-jdk15on = 1.60
-org.glassfish.jersey.*:* = 2.22.2
+org.glassfish.jersey.*:* = 2.25.1
```

Comment:
Apparently 2.26.0 onwards contains breaks (package renames), so this is the highest version that we can use: https://mvnrepository.com/artifact/org.glassfish.jersey.core/jersey-server